Powershell-PoC-Environment-Setup.ps1

```diff
@@ -495,23 +708,33 @@ Set-AzureRmVirtualNetworkSubnetConfig -Name $AppSNNameEast -VirtualNetwork $vnet
 #-------------------------------------------------------
 
 Write-Verbose "Create Storage Account for East Region & West Region" -Verbose
+Write-WithTime -Output "Create Storage Account for East Region & West Region" -Log $Log
 
 # Storage Account Names must be unique - make sure to change it here
-$saWestName = "<storage account name>"
+# ORIGINAL: $saWestName = "<storage account name>"
+# NEW: 0.00.00.0019
+# Create a new random string, then extract the 4 digits to use as the last characters for the storage account name for each region
+New-RandomString

```

Comment:
Inside the New-RandomString function, the script scope variable is already assigned to $randomString.
 [string]$Script:randomString = $randomStringWithSpaces.Replace(" ","")